apps/condo/domains/miniapp/schema/SendAppPushMessageService.test.js

```diff
@@ -246,4 +255,42 @@ describe('SendAppPushMessageService', () => {
             })
         })
     })
+
+    describe.skip('real notification send for debugging', () => {
```

Comment:
Any comment why skip?

Reply:
it's for debugging to run locally. n need to run it on cicd